src/network/networkd-route.c

```diff
@@ -718,6 +746,8 @@ int network_add_ipv4ll_route(Network *network) {
         n->family = AF_INET;
         n->dst_prefixlen = 16;
         n->scope = RT_SCOPE_LINK;
+        n->scope_set = true;
+        n->table_set = true;
```

Comment:
Do we require these two variables ? Can't they set up unset like -1.

Reply:
scope or table can be `(unsigned) -1`...